.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,15 @@ resolves #1235
 ### This pullrequest changes
 
 <!-- Please describe what your pullrequest is changing -->
+
+<cut/>
+
+### Pull Request Readiness Checklist
+
+- [ ] I agree to contribute to the project under OpenCV ([BSD](https://github.com/opencv/opencv/blob/master/LICENSE)) license
+- [ ] The PR is proposed to proper branch ([details](https://github.com/opencv/opencv/wiki/How_to_contribute#before-you-start-contributing-you-should))
+- [ ] There is reference to original bug report and related work
```

Comment:
=> "If the patch fixes a submitted bug, or implements a requested feature, the corresponding issue should be referenced"